src/compiler/utilities.ts

```diff
@@ -1966,7 +1966,7 @@ function isShorthandAmbientModule(node: Node | undefined): boolean {
 }
 
 /** @internal */
-export function isBlockScopedContainerTopLevel(node: Node): boolean {
+export function isBlockScopedOrTopLevelContainer(node: Node): boolean {

```

Comment:
I thought I'd have to use this, I didn't end up using it but I think the new name is better so I left this change here